prober/http.go

```diff
@@ -397,10 +400,13 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 	request = request.WithContext(ctx)
 
 	for key, value := range httpConfig.Headers {
-		if strings.Title(key) == "Host" {
+		normalizedKey := strings.Title(key)
```

Comment:
Why this indirection?

Reply:
ah, that's a leftover from a different implementation, thanks for catching it